src/wp-consent-api.js

```diff
@@ -53,7 +53,7 @@ function wp_has_consent(category) {
  * @param {string} name The cookie name to set.
  * @param {string} value The cookie value to set.
```

Comment:
Let's make sure there's a docblock parameter for the new variable and then this is good to go.

Reply:
Ahh, forgive my rush. I did add the additional param to doc block in this commit: https://github.com/humanmade/consent-api-js/pull/10/commits/779b488e3c8173e3f2da140a1ff397d58750412c

Reply:
> This is probably something we could also copy upstream to https://github.com/rlankhorst/wp-consent-level-api -- would you like to create a PR for that or do you want me to? (I'm fine either way.)

I'm happy to do this.